File: src/main/java/com/enzo/testemuralis/models/Contato.java
package com.enzo.testemuralis.models;

import com.enzo.testemuralis.models.dto.ContatoRequestDTO;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = Contato.TABLE_NAME)
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(of = "id")
public class Contato {
    public static final String TABLE_NAME = "contato";

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, length = 100)
    private String tipo;

    @Column(nullable = false, length = 100)
    private String texto;

    public Contato(ContatoRequestDTO contatoRequestDTO) {
        this.tipo = contatoRequestDTO.tipo();
        this.texto = contatoRequestDTO.texto();
    }
}

File: dotnet/TesteMuralis/Models/Contato.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TesteMuralis.Dto;

namespace TesteMuralis.Models
{
    public class Contato
    {
        public int Id { get; set; }
        public string Tipo { get; set; }
        public string Texto { get; set; }

        public Contato() {}

        public Contato(ContatoRequestDTO contato) {
            Tipo = contato.Tipo;
            Texto = contato.Texto;
        }
    }
}